test/config.ini.in

```diff
@@ -6,6 +6,7 @@
 # test/functional/test_runner.py and test/util/bitcoin-util-test.py
 
 [environment]
+PACKAGE_NAME=@PACKAGE_NAME@
```

Comment:
You probably need to add something here, though I'm not sure what:
https://github.com/bitcoin/bitcoin/blob/master/.appveyor.yml#L41-L48

Reply:
Update, indeed, see #15903